pkg/controllers/user/cis/clusterScanHandler.go

```diff
@@ -84,14 +85,15 @@ func (csh *cisScanHandler) Create(cs *v3.ClusterScan) (runtime.Object, error) {
 	if !v3.ClusterScanConditionCreated.IsTrue(cs) {
 		logrus.Infof("cisScanHandler: Create: deploying helm chart")
 		currentK8sVersion := cluster.Spec.RancherKubernetesEngineConfig.Version
+		shortK8sVersion := util.GetTagMajorVersion(currentK8sVersion)
 		cisConfigParams, err := kontainerdrivermetadata.GetCisConfigParams(
-			currentK8sVersion,
+			shortK8sVersion,
```

Comment:
I am not seeing what this changes outside of where util.GetTagMajorVersion is getting called? Is this a refactor to get the debug log to have more information?

Reply:
Earlier, `GetCisConfigParams` was figuring out the k8sversion, but now it's just doing a get. Hence, the "default" is fetched with the change in logic.

Reply:
But the value `name` being passed into the `GET` is the same in both versions

Reply:
https://github.com/rancher/rancher/pull/25183/files#diff-2bb8b7c6b4990566a18c3e0a0f28c683R98

Reply:
The value "default" was being used to get the short version and it would return an empty string, which leads to fetching the CRD without a name and the call craps out with the error mentioned in the original issue.

Reply:
Ohh this is for the error case where the first call fails then `default` is needed to be passed and not ran through GetTagMajorVersion. Some go tests would really help out here.

Reply:
yes, so if we have a cluster with a k8s version that CIS is not aware of, it would use "default"